internal/httpserve/route/static.go

```diff
@@ -148,3 +148,26 @@ func registerAppleMerchantHandler(router *Router) (err error) {
 
 	return nil
 }
+
+// registerExampleCSVHandler serves up the text output of the example csv file
+func registerExampleCSVHandler(router *Router) (err error) {
+	path := "/example/csv"
+	method := http.MethodPost
+	name := "ExampleCSV"
+
+	route := echo.Route{
+		Name:        name,
+		Method:      method,
+		Path:        path,
+		Middlewares: authMW,
+		Handler: func(c echo.Context) error {
+			return router.Handler.ExampleCSV(c)
```

Comment:
You need to set the response header here; otherwise its using the default from the `mime` middleware:

```suggestion
			c.Response().Header().Set(httpsling.HeaderContentType, "text/csv")

			return router.Handler.ExampleCSV(c)
```